docs/source/installation/index.rst

```diff
@@ -75,9 +75,9 @@ following::
 Within the directory returned, create a file named ``pds_doi_service.ini``.
 
 In this file you can override any option set in the default configuration file
-``pds_doi_service/core/util/conf.ini.default`` found within the package. An
+``pds_doi_service/core/util/conf.default.ini`` found within the package. An
 example of this file may be found
-`here <https://raw.githubusercontent.com/NASA-PDS/pds-doi-service/main/src/pds_doi_service/core/util/conf.ini.default>`_.
+`here <https://raw.githubusercontent.com/NASA-PDS/pds-doi-service/main/src/pds_doi_service/core/util/conf.default.ini>`_.
```

Comment:
Nit pick: [don't](https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/) [hyperlink](https://heyoka.medium.com/dont-use-click-here-f32f445d1021) [the](https://ux.stackexchange.com/questions/12100/why-shouldnt-we-use-words-such-as-here-and-this-in-textlinks) [word](https://www.wordpress-web-designer-raleigh.com/2015/04/4-reasons-to-avoid-using-click-here-in-link-text/) [here](https://www.w3.org/QA/Tips/noClickHere) 😅

(Feel free to ignore this)

Reply:
Temporarily ignoring, but will go back and fix that